src/main/java/org/schemaspy/view/HtmlRoutinesPage.java

```diff
@@ -61,16 +61,16 @@ public void write(Database db, LineWriter html) throws IOException {
             write(routine, html);
         }
 
-        writeFooter(html);
+        //writeFooter(html);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)